cmake/SourceFileList.cmake

```diff
@@ -135,7 +134,6 @@ set(eoserv_ALL_SOURCE_FILES
 	src/hashupdater.hpp
 	src/i18n.cpp
 	src/i18n.hpp
-	src/main.cpp
```

Comment:
Where do these main files get picked up by the build system now that they are removed from this? (Admittedly, it's late here and I didn't go search for them. Just curious)

Reply:
They're moved to the `CMakeLists.txt` file. These file lists essentially become an "everything but main" list of files, so that they can be linked across `eoserv` and `eoserv_test` targets (if there's a better way to do this please let me know)